lib/tanker/crypto.rb

```diff
@@ -13,6 +13,10 @@ def self.generichash(input, size)
       RbNaCl::Hash.blake2b(binary_input, digest_size: size)
     end
 
+    def self.hashed_provisional_value(value)
+      Base64.strict_encode64(Crypto.generichash(value, Identity::BLOCK_HASH_SIZE))
```

Comment:
It's a bit weird to depend on `Identity` here (circular), we should probably move the `BLOCK_HASH_SIZE` constant here to avoid that.

Only `Identity` will depend on `Crypto` then.

Reply:
Good point, moved it there along with the hash_user_id function